physical/consul.go

```diff
@@ -154,6 +154,10 @@ func newConsulBackend(conf map[string]string, logger log.Logger) (Backend, error
 
 	// Configure the client
 	consulConf := api.DefaultConfig()
+	// Set MaxIdleConnsPerHost to the number of processes used in expiration.Restore
+	tr := cleanhttp.DefaultPooledTransport()
+	tr.MaxIdleConnsPerHost = 64
```

Comment:
Currently they are in two different packages with cyclic imports, so I can't use `RestoreWorkerCount`. let me think about how to make this a bit better.